### DISCUSSION TITLE
Multi-GPU Supported!

### DISCUSSION THREAD

**Discussion by crystian** [crystian/ComfyUI-Crystools#22] [Show and tell]
A lot of changes, but it is here!

**Comment by lord-lethris** [comment]
Regarding this - What do you mean by Multi-GPU Support?
Is that if you have more than one GPU - OR, does this mean Crystools supports more than one GPU type?

I'm asking because I get this error when loading up ComfyUI:

```
[Crystools INFO] Crystools version: 1.12.0
[Crystools INFO] CPU: AMD Ryzen 9 5900X 12-Core Processor - Arch: AMD64 - OS: Windows 10
[Crystools ERROR] Could not init pynvml.NVML Shared Library Not Found
[Crystools WARNING] No GPU with CUDA detected.
```

And I'm fairly certain that this is because I'm running an AMD GPU & DirectML.

Would be nice if the resource bars supported AMD GPU / DirectML load stats.

**Comment by crystian** [comment]
more than one nvidia video card ¯\_(ツ)_/¯
